woocommerce/woocommerce-android
# [Woo POS] Empty/half hidden cart state

Closes: #11714


### Description

* Fixes margins between subscreens
* Adds empty cart state

### Steps to reproduce


### Testing information

* Add/remove items to the cart. Notice the new state with animation
* Notice changed margin between subscreens

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/5a83e1b5-e9fc-4609-aa70-307fbcc64087
